src/functions/software-registry/update-software/handler.ts

```diff
@@ -0,0 +1,60 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { DynamoDB } from 'aws-sdk';
+import { middyfy } from 'src/libs/lambda';
+
+const dynamoDb = new DynamoDB.DocumentClient();
+const tableName = process.env.DYNAMODB_TABLE;
+
+export const updateSoftwareHandler: APIGatewayProxyHandler = async (event) => {
+    const data = JSON.parse(event.body);
+
+    if (!data.name) {
```

Comment:
If name is required it should be described as such in the spec.
Is it the only attribute we need to validate here?